duckduckgo/privacy-essentials-safari
# fix project file

Task/Issue URL: n/a
Tech Design URL:
CC:

**Description**:

Project file got messed up when merging after the last release.

**Steps to test this PR**:
1. Check out the code
3. Check Xcode doesn't have any "red" files / folders / groups 
1. Build the app, ensure it runs, unit tests pass, etc


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
